extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -465,20 +539,36 @@ function _processTID1410Measurement(mergedContentSequence) {
 
   NUMContentItems.forEach(item => {
     const { ConceptNameCodeSequence, MeasuredValueSequence } = item;
-
     if (MeasuredValueSequence) {
       measurement.labels.push(
         _getLabelFromMeasuredValueSequence(ConceptNameCodeSequence, MeasuredValueSequence)
       );
     }
   });
 
+  const findingSites = mergedContentSequence.filter(
+    item =>
+      item.ConceptNameCodeSequence.CodingSchemeDesignator === CodingSchemeDesignators.SCT &&
```

Comment:
This is inconsistent with other uses of the labels/finding sites.  Labels is a bit artificial - I'd rather see it use the finding sites directly.  I can show you an example of how that works in the basic test view.

Reply:
I'm using the same code today but I extended it to support SCT instead of just SRT.